agent/api/task.go

```diff
@@ -72,11 +72,11 @@ type Task struct {
 	Containers []*Container
 	// Volumes are the volumes for the task
 	Volumes []TaskVolume `json:"volumes"`
-	// VCPULimit is a task-level limit for compute resources. A value of 1 means that
```

Comment:
I'm okay with leaving VCPULimit and MemoryLimit on our internal model

Reply:
I can revert back to this if you feel strongly about it.